lib/ddtrace/span.rb

```diff
@@ -65,8 +65,8 @@ def get_tag(key)
       @meta[key]
     end
 
-    # Set the given key / value metric pair on the span. Keys must be string.
-    # Values must be floating point numbers.
+    # This method sets a tag for a floating point value for the given key. It acts
```

Comment:
`for a floating point` -> `with a floating point`. Wondering. It's just about language/wording, indeed it's good not to advocate too much for that `set_metric` usage when there's almost no difference with `set_tag`.